Отчёты Allure

agassshhhiii opened the pull request:
При выполнении команды "gradle test allureReport" выводится такая ошибка (скрин), но отчёты создались. Не совсем понимаю, правильные ли отчеты и должна ли быть такая ошибка.
![изображение_2025-01-30_135502014](https://github.com/user-attachments/assets/21570f9a-948d-4d96-bf26-9593560a11f4)

agassshhhiii committed:
Отчёты Allure

dderemina commented:
Так, а ты в отчёт заглядывала? Там корректная информация о твоём прогоне? 
Вообще ты после последних изменений пробовала запускать тесты, всё ОК?
Мне бы хотелось, чтобы ты сама попробовала разобраться, в чём проблема)
Во второй строке у тебя есть отчёт с логом ошибки. Мне в него не заглянуть.
Могу предположить, что у тебя какое-то расхождении версии джавы, которую ты используешь в проекте, и версии джавы, которую поддерживают новые плагины и библиотеки, используемые в проекте.
Попробуй повысить версию плагина аллюра.

agassshhhiii committed:
Изменила версию allure на 2.12.0.
Нашла решение, где был изменён путь до проекта, попробовала - заработало 0.0
Подгружаю изменённые файлы

agassshhhiii commented on allure-results/0186e437-83a2-46d4-82b2-ecd2f8b3812a-container.json:
И что на счет ветки с кодом? Всё ли нормально?
